### PR TITLE
feat(plasma-core): custom width/height for TextArea

### PR DESCRIPTION
**Pull request by kayman233**
Добавилась возможность пробрасывать `height` и `width` для компонента `TextArea` во всех пакетах.
Также появилась возможность использовать атрибуты `rows` и `cols`.

Пример c  `height` и `width`:
``` js
<TextArea
    value="Value"
    placeholder="Placeholder"
    helperText="Helper text"
    width="10rem"
    height="10rem"
    contentRight={<IconEye color="inherit" size="s" />}
/>
```
Снимок экрана 2023-07-04 в 16 41 20

:baby_chick: Download canary assets:

[color_asdk_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_ios-swift--canary.593.5468287604.swift)  
[color_asdk_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_kotlin--canary.593.5468287604.kt)  
[color_asdk_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_react-native--canary.593.5468287604.ts)  
[color_asdk_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_xml--canary.593.5468287604.xml)  
[color_plasma_b2c_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.593.5468287604.swift)  
[color_plasma_b2c_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.593.5468287604.kt)  
[color_plasma_b2c_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.593.5468287604.ts)  
[color_plasma_b2c_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.593.5468287604.xml)  
[color_sberHealth_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.593.5468287604.swift)  
[color_sberHealth_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.593.5468287604.kt)  
[color_sberHealth_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.593.5468287604.ts)  
[color_sberHealth_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.593.5468287604.xml)  
[color_sbermarket_business_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.593.5468287604.swift)  
[color_sbermarket_business_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.593.5468287604.kt)  
[color_sbermarket_business_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.593.5468287604.ts)  
[color_sbermarket_business_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.593.5468287604.xml)  
[color_sbermarket_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.593.5468287604.swift)  
[color_sbermarket_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.593.5468287604.kt)  
[color_sbermarket_metro_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.593.5468287604.swift)  
[color_sbermarket_metro_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.593.5468287604.kt)  
[color_sbermarket_metro_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.593.5468287604.ts)  
[color_sbermarket_metro_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.593.5468287604.xml)  
[color_sbermarket_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.593.5468287604.ts)  
[color_sbermarket_selgros_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.593.5468287604.swift)  
[color_sbermarket_selgros_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.593.5468287604.kt)  
[color_sbermarket_selgros_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.593.5468287604.ts)  
[color_sbermarket_selgros_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.593.5468287604.xml)  
[color_sbermarket_wlbusiness_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.593.5468287604.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.593.5468287604.kt)  
[color_sbermarket_wlbusiness_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.593.5468287604.ts)  
[color_sbermarket_wlbusiness_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.593.5468287604.xml)  
[color_sbermarket_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.593.5468287604.xml)  
[color_sberonline_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.593.5468287604.swift)  
[color_sberonline_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.593.5468287604.kt)  
[color_sberonline_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.593.5468287604.ts)  
[color_sberonline_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.593.5468287604.xml)  
[color_sberprime_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.593.5468287604.swift)  
[color_sberprime_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.593.5468287604.kt)  
[color_sberprime_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.593.5468287604.ts)  
[color_sberprime_xml--canary.593.5468287604.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.593.5468287604.xml)  
[shadow_sbermarket_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.593.5468287604.ts)  
[typo_mage_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.593.5468287604.swift)  
[typo_mage_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.593.5468287604.kt)  
[typo_mage_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.593.5468287604.ts)  
[typo_plasma_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.593.5468287604.swift)  
[typo_plasma_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.593.5468287604.kt)  
[typo_plasma_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.593.5468287604.ts)  
[typo_ruler_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.593.5468287604.swift)  
[typo_ruler_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.593.5468287604.kt)  
[typo_ruler_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.593.5468287604.ts)  
[typo_sage_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.593.5468287604.swift)  
[typo_sage_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.593.5468287604.kt)  
[typo_sage_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.593.5468287604.ts)  
[typo_sbermarket_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.593.5468287604.swift)  
[typo_sbermarket_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.593.5468287604.kt)  
[typo_sbermarket_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.593.5468287604.ts)  
[typo_soulmate_ios-swift--canary.593.5468287604.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.593.5468287604.swift)  
[typo_soulmate_kotlin--canary.593.5468287604.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.593.5468287604.kt)  
[typo_soulmate_react-native--canary.593.5468287604.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.593.5468287604.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.219.0-canary.593.5468287604.0
  npm install @salutejs/plasma-core@1.124.0-canary.593.5468287604.0
  npm install @salutejs/plasma-hope@1.219.0-canary.593.5468287604.0
  npm install @salutejs/plasma-icons@1.153.0-canary.593.5468287604.0
  npm install @salutejs/plasma-temple@1.168.0-canary.593.5468287604.0
  npm install @salutejs/plasma-ui@1.200.0-canary.593.5468287604.0
  npm install @salutejs/plasma-web@1.219.0-canary.593.5468287604.0
  npm install @salutejs/plasma-sb-utils@0.122.0-canary.593.5468287604.0
  # or 
  yarn add @salutejs/plasma-b2c@1.219.0-canary.593.5468287604.0
  yarn add @salutejs/plasma-core@1.124.0-canary.593.5468287604.0
  yarn add @salutejs/plasma-hope@1.219.0-canary.593.5468287604.0
  yarn add @salutejs/plasma-icons@1.153.0-canary.593.5468287604.0
  yarn add @salutejs/plasma-temple@1.168.0-canary.593.5468287604.0
  yarn add @salutejs/plasma-ui@1.200.0-canary.593.5468287604.0
  yarn add @salutejs/plasma-web@1.219.0-canary.593.5468287604.0
  yarn add @salutejs/plasma-sb-utils@0.122.0-canary.593.5468287604.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-593/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-593/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-593/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-593/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-593/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-593/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-593/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-593/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-593/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-593/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-593/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-593/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-593/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-593/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-593/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-593/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-593/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-593/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: packages/plasma-core/src/components/TextArea/TextArea.tsx [hunk 1]
```diff
@@ -9,21 +9,36 @@ export interface TextAreaProps extends Omit<FieldProps, 'contentLeft'>, Textarea
      * Изменение размера текстового поля.
      */
     resize?: TextareaResize;
+    /**
+     * Высота текстового поля.
```
Comment (Yakutoc) [inline packages/plasma-core/src/components/TextArea/TextArea.tsx:13]: @kayman233 Может укажем сразу, что значение в `rem` указывается?

File: packages/plasma-hope/src/components/TextArea/views/b2c/TextAreaB2C.tsx [hunk 2]
```diff
@@ -17,18 +17,19 @@ import type { TextAreaPropsCommon } from '../../types';
 import { FieldWrapper, FieldHelper, FieldHelpers, applyInputStyles } from './Field'; // INFO: Этот компонент скопирован из plasma-b2c
 import { textAreaProps } from './TextArea.props';
 
-const StyledTextArea = styled(BaseArea)<{ $isHelper: boolean }>`
+const StyledTextArea = styled(BaseArea)<{ $isHelper: boolean; height?: string; rows?: number }>`
     ${applyInputStyles};
 
-    --computed-height: calc(9.375rem - var(--field-helpers-padding-top) - var(--padding-bottom) - 14px);
+    --computed-height: calc(var(--field-helpers-padding-top) + var(--padding-bottom) + 14px);
```
Comment (Yakutoc) [inline packages/plasma-hope/src/components/TextArea/views/b2c/TextAreaB2C.tsx:23]: [Уточнение]

А почему изменились операции с минуса на плюс? )

File: packages/plasma-ui/src/components/TextArea/TextArea.tsx [hunk 3]
```diff
@@ -14,12 +14,18 @@ const StyledTextArea = styled(BaseArea).attrs(spatnavClassNameAttrs)`
     border-radius: 1rem;
 `;
 
+const StyledFieldContentWrapper = styled.div<{ width?: string }>`
+    position: absolute;
```
Comment (Yakutoc) [inline packages/plasma-ui/src/components/TextArea/TextArea.tsx:18]: [Уточнение]

Можешь в двух словах рассказать почему через `absolute`?

Comment (kayman233) [reply packages/plasma-ui/src/components/TextArea/TextArea.tsx:18]: Ну у нас здесь по аналогии с web/b2c для того, чтобы contentRight был привязан по ширине к контейнеру

File: packages/plasma-hope/src/components/TextArea/views/b2c/TextAreaB2C.tsx (hunk 2 above)
Comment (kayman233) [reply packages/plasma-hope/src/components/TextArea/views/b2c/TextAreaB2C.tsx:23]: возможно стоит как-то по-другому переименовать. Но у нас высота несколькими строчками ниже вычисляется, используя эту переменную, но теперь вместо 9.375rem может быть произвольная пользовательская, ну а минус на плюс просто из-за арифметики вычитания

File: packages/plasma-core/src/components/TextArea/TextArea.tsx (hunk 1 above)
Comment (kayman233) [reply packages/plasma-core/src/components/TextArea/TextArea.tsx:13]: в принципе можно, тогда вообще наверное лучше number передавать?

File: packages/plasma-core/src/components/TextArea/TextArea.tsx [hunk 4]
```diff
@@ -9,21 +9,36 @@ export interface TextAreaProps extends Omit<FieldProps, 'contentLeft'>, Textarea
...
 
-    ${({ resize }) =>
+    height: ${({ height, rows }) => (rows ? 'unset' : height || '9.375rem')};
+    width: ${({ width, cols }) => (cols ? 'unset' : width || '100%')};
+
+    ${({ resize, width, cols }) =>
         css`
             ${resize && `resize: ${resize};`}
-            ${resize !== 'both' && resize !== 'horizontal' && 'min-width: 100%;max-width: 100%;'}
+            ${resize !== 'both' &&
```
Comment (Yeti-or) [inline packages/plasma-core/src/components/TextArea/TextArea.tsx:39]: тут конечно ничего не ясно =/

**Review by Yeti-or — APPROVED** [review]

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-593/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-593/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-593/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-593/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-593/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-593/temple-storybook/

File: packages/plasma-ui/src/components/TextArea/TextArea.tsx [hunk 5]
```diff
@@ -34,8 +40,20 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(function
...
+                ref={ref}
+                id={id}
+                placeholder={placeLabel}
+                disabled={disabled}
+                status={status}
+                width={width}
+                {...rest}
+            />
+            {contentRight && (
+                <StyledFieldContentWrapper width={width}>
```
[the remark this answers is not in the document]
Comment (kayman233) [reply packages/plasma-ui/src/components/TextArea/TextArea.tsx]: а вот оно в принципе было сломано, там один тест был с некорректным снапшотом

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-593/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-593/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-593/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-593/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-593/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-593/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out report in job artifacts!

Top 5 significant render changes:
packages/plasma-ui/src/components/TextArea/TextArea.perftest.tsx#Default: 49.95%

File: website/plasma-ui-docs/docs/components/TextArea.mdx [hunk 6]
```diff
@@ -32,6 +32,8 @@ export function App() {
 }
 ```
 
+Также можно регулировать высоту и ширину, используя свойства `height` и `width`, указав значения в `rem`.
+
```
Comment (neretin-trike) [inline website/plasma-ui-docs/docs/components/TextArea.mdx:36]: Тут можно добавить "или соответствующие свойстваи css значения"

File: website/plasma-web-docs/docs/components/TextArea.mdx [hunk 7]
```diff
@@ -32,6 +32,8 @@ export function App() {
 }
 ```
 
+Также можно регулировать высоту и ширину, используя свойства `height` и `width`, указав значения в `rem`.
+
```
Comment (neretin-trike) [inline website/plasma-web-docs/docs/components/TextArea.mdx:36]: И тут

File: packages/plasma-hope/src/components/TextArea/views/b2c/TextAreaB2C.tsx (hunk 2 above)
Comment (neretin-trike) [reply packages/plasma-hope/src/components/TextArea/views/b2c/TextAreaB2C.tsx:25]: А можешь попробовать заменить на rem вместо 14px?

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-593/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-593/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-593/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-593/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-593/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out report in job artifacts!

Top 5 significant render changes:
packages/plasma-ui/src/components/TextArea/TextArea.perftest.tsx#Default: 45.45%

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-593/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-593/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-593/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-593/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-593/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-593/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Pull request merged**